config/routes.rb

```diff
@@ -24,7 +24,7 @@
         :as => :show_picture
   get '/pictures/:id/zoom/:name.:format' => 'pictures#zoom',
         :as => :zoom_picture
-  get "/pictures/:id/thumbnails/:size(/:crop)(/:crop_from/:crop_size)/:name.:format" => 'pictures#thumbnail',
+  get "/pictures/:id/thumbnails(/:size)(/:crop)(/:crop_from/:crop_size)/:name.:format" => 'pictures#thumbnail',
```

Comment:
Line is too long. [111/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.